packages/react-next/etc/react.api.md

```diff
@@ -0,0 +1,12 @@
+## API Report File for "@fluentui/react"

```

Comment:
Why is this here?

Reply:
This is what API extractor generates. It fails without this. This will fill up with things once we've added code to this package.

Reply:
What I meant is that there are API review files for *both* `react` and `react-next`, which seems weird.

Reply:
oh weird! Thanks!